repos/spack_repo/builtin/packages/at_spi2_core/package.py

```diff
@@ -42,6 +42,7 @@ class AtSpi2Core(MesonPackage):
     depends_on("libxi")
     depends_on("libxml2", type="build", when="@2.47:")
     depends_on("libxtst")
+    depends_on("libxml2")
```

Comment:
Is this intentional?
```suggestion
    depends_on("libxml2")
    depends_on("libxtst")
```

Reply:
It happened so long ago I am not 100% sure, but I think there was a probably linking error around this at some point? Otherwise it is a bad rebase, which based on how long lived these change have been it is entirely possible.